components/textfield/stories/template.js

```diff
@@ -234,10 +234,6 @@ export const TextFieldGroup = Variants({
 		labelText: "Username",
 		value: "UsernameWiderThanInput@ReallyLongEmail.com"
 	},
-	{
-		testHeading: "With placeholder",
-		placeholder: "Username",
```

Comment:
I don't think there's much reason to test the placeholder here, so I removed it after initially adding it.

I did leave it in Storybook as a control but I'm open to hearing an argument for removing it.